parlai/zoo/multimodal_blender/README.md

```diff
@@ -0,0 +1,145 @@
+--update-freq 2 \
+--history-add-global-end-token end \
+--delimiter '  ' \
+--lr-scheduler-patience 3 \
+--warmup-updates 100 \
+--multitask-weights stochastic \
+--relu-dropout 0.0 \
+--save-after-valid True \
+--skip-generation True \
+-lr 1e-06 \
+-veps 0.25 \
+-vmm min \
+-vmt ppl \
+-vp 10 \
+--init-model ${DOMAIN_PRETRAINED_MODEL_PATH} \
+--image-fusion-type early \
+--n-segments 2 \
+--n-image-channels 100 \
+--model-file ${FINETUNED_MODEL_PATH}
+```
+
+## Inference
+After fine-tuning, run the following command to interact with your model:
+```
+python parlai/scripts/safe_interactive.py \
+-t blended_skill_talk \
+-mf ${FINETUNED_MODEL_PATH} \
+--model projects.multimodal_blender.agents:BiasAgent \
+--delimiter $'\n' \
+--beam-block-ngram 3 \
+--beam-context-block-ngram 3 \
+--beam-min-length 20 \
+--beam-size 10 \
+--inference beam \
+--model-parallel False
+```
+
+## Safety
+We have undertaken several steps to make this bot safer. The original BlenderBot model was released with a `safe_interactive.py` script using a state-of-the-art safety classifier system on user and bot messages, as discussed [here](https://parl.ai/projects/recipes/), and we do so here as well. In *this* model, we additionally incorporate several strategies not found in previous systems:
+
+1. The bot was trained to distinguish between utterances containing gendered words and utterances without gendered words, and during inference, the bot will be set to respond with utterances without gendered words, to reduce potential gender bias. 
+   - Compared to a bot not trained to be sensitive to gendered words, this bot reduces the frequency of utterances with male words by a factor of 9 and with female words by a factor of 4.
+2. When responding to an image, the bot was trained to distinguish between utterances categorized as “positive” or “neutral” in tone and utterances categorized as “negative”. During inference, the bot is able to respond to an image with only “positive” or “neutral” utterances given the appropriate context string.
+    - We find that, in general, “positive” or “neutral” utterances are much less likely to be classified as containing offensive language than “negative” utterances.
+
+Even with these steps, however, we have not addressed all possible safety concerns with this model, and we do not make any kind of guarantee that the model will not produce unsafe or offensive responses. (For instance, even with these measures, 10% of responses given examples from the Image-Chat validation set are still flagged as offensive by the safety classifier presented in [the “Build it break it fix it for dialogue safety” paper of Dinan et al., 2019](https://arxiv.org/abs/1908.06083). However, in this case we have not incorporated the safety features of the `safe_interactive.py` script described above, where the safety classifier mitigates a number of these concerns -- but not all.) If you choose to train or use this model, you do so entirely at your own risk.
```

Comment:
we can probably remove the parentheses around the second/third sentence

Reply:
Hmm how about keeping it around the 3rd sentence only? That sentence *is* a true parenthetical - I don't want the reader to think that the safety features of `safe_interactive.py` are unusable for our model, for example

Reply:
actually, after re-reading, happy to keep the full parenthetical. i was missing the forest for the trees